### PR TITLE
extensions/ext: Add VK_EXT_hdr_metadata extension

### PR DESCRIPTION
**Pull request by MarijnS95**

### CODE REVIEW

File: ash/src/extensions/ext/mesh_shader.rs
Comment (Ralith) [inline ash/src/extensions/ext/mesh_shader.rs:1]: Unrelated change?

**Review by Ralith — APPROVED** [review]

File: ash/src/extensions/ext/mesh_shader.rs
Comment (MarijnS95) [reply ash/src/extensions/ext/mesh_shader.rs:1]: It's intentional: I used this mod as a base and noticed that `;` was hampering our ability to ensure that the return types are identical.
